tox.ini

```diff
@@ -9,6 +9,7 @@ deps =
     nose
     nose-exclude
     unittest2
+    -egit+https://github.com/google/oauth2client.git#egg=oauth2client
```

Comment:
Sorry I am a n00b, what is WNTL?

And yes, we'll need a pin once it gets released.